### ISSUE TITLE
problems following readme

### ISSUE THREAD

**Issue by mk270** [OpenBookPublishers/obp-style#1]
```
mk270@savile ~/Src/obp-style> docker run --rm -v $(pwd)/dist/:/src/dist openbookpublishers/obp-style
Unable to find image 'openbookpublishers/obp-style:latest' locally
docker: Error response from daemon: manifest for openbookpublishers/obp-style:latest not found: manifest unknown: manifest unknown.
See 'docker run --help'.
```

Is this my fault or the README's fault or neither or both?

**Comment by ja573** [comment]
It was the README, the image's not been pushed to the hub. Fixed with b2c4d0acc393c3a9107211003af3cbbb4ce41d1b

**Comment by mk270** [comment]
thanks - wasn't really sure what the error message meant, but to be fair it did sort of say what the problem was
